go/logic/migrator.go

```diff
@@ -401,6 +406,10 @@ func (this *Migrator) Migrate() (err error) {
 	}
 	this.printStatus(ForcePrintStatusRule)
 
+	if this.migrationContext.CountTableRowsCancelFunc != nil {
```

Comment:
Needs to be thread protected. Potential concurrent change: https://github.com/github/gh-ost/pull/846/files#diff-5aa50dde7ca1baf7a819eecf740ea220R543